ui/logviewer.html

```diff
@@ -41,10 +41,12 @@
 
           <!-- Raw log button -->
           <li>
-            <a title="Open the raw log in a new window"
+            <a title="{{logError ? 'Raw log link no longer exists or has expired (click for path)' :
```

Comment:
These curly braces around this this ternary aren't necessary (based on a highly upvoted S.O.) so I will remove them. I also opened a (hopefully valid) issue 12581 against angular.js to improve their documentation in this area.